website/thaliedje/templates/thaliedje/search.html

```diff
@@ -4,11 +4,14 @@
 <div id="player-search-container-{{ player.id }}">
     <input v-model="query" id="search" type="text" maxlength="100" placeholder="Search for something to play" class="container-fluid form-control mb-3"/>
     <div id="results" class="px-3 search-list">
-        <div class="d-flex flex-column">
-            <template v-if="tracks.length > 0 && query !== ''">
+        <div class="row">
+            <div class="col-lg-4" v-if="tracks.length > 0 && query !== ''">
```

Comment:
This multi column layout should only be used for people that have permission to search for playlists and albums. For regular users, there should just be 1 column. Or at least the columns should be centered
<img width="1131" alt="image" src="https://user-images.githubusercontent.com/7915741/181231891-a8c1e460-c2de-4b2f-911b-554e1dc11f1e.png">

Reply:
For non-privileged users the modal can perhaps also be less wide